src/software/ai/hl/stp/tactic/attacker/attacker_fsm.cpp

```diff
@@ -62,20 +62,21 @@ void AttackerFSM::keepAway(const Update& event,
                   << "to enemy goal center.";
     }
 
-    auto keepaway_dribble_dest =
-        findKeepAwayTargetPoint(*event.common.world_ptr, best_pass_so_far);
+    auto keepaway_dribble_dest = findKeepAwayTargetPoint(
```

Comment:
Why not `Point` instead of `auto`?

Reply:
It's an arbitrary decision and shouldn't make any difference to the code logic. `Point` is probably better to use in scenarios like this, as it explicitly shows the reader what the function returns. But in this case I kept the existing code that was there.